Objects/longobject.c

```diff
@@ -6020,22 +6020,12 @@ int_bit_count_impl(PyObject *self)
 /*[clinic input]
 int.as_integer_ratio
 
-Return integer ratio.
-
-Return a pair of integers, whose ratio is exactly equal to the original int
-and with a positive denominator.
-
->>> (10).as_integer_ratio()
```

Comment:
> I don't think there is anything wrong with it per say

But after "return the tuple (int(x), 1)"? :)

Reply:
> I think it's best to probably just leave it untouched and to keep this PR focused on the issue at hand

Agreed with this. I'd keep the original wording, too - it accurately describes the intent rather than the implementation (and the `(int(x), 1)` is awkward.) IOW, I think we could just leave `Objects/longobject.c` unaltered.

Reply:
> I'd keep the original wording, too - it accurately describes the intent

Unfortunately, no:)  Here is how it looks in the main:
```
    Return integer ratio.

    Return a pair of integers, whose ratio is exactly equal to the original int
    and with a positive denominator.

    >>> (10).as_integer_ratio()
    (10, 1)
    >>> (-10).as_integer_ratio()
    (-10, 1)
    >>> (0).as_integer_ratio()
    (0, 1)
```
It doesn't mention that the ratio is in lowest terms.  The original patch was only with this little correction.

I'll revert to this, but IMHO the current version is much more obvious: "hey, we just return components of the canonicalized fraction for a given integer".